### PR TITLE
VideoPress: Change hover preview loop duration component to Timestamp control and fix default values

### LINKED ISSUE #29809: VideoPress: use TimestampControl for PreviewOnHover duration

### PR DESCRIPTION
**Pull request by dhasilva**
Fixes #29809

## Proposed changes:

* Changes the  `NumberControl` component to `TimestampControl` for the loop duration
* Adds two decimal places to the selectors

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to the block editor
* Add a VideoPress video block
* Go to the `Poster and review` panel
* Check the `Video preview on hover` toggle
* Check that the loop duration uses a TimestampControl component
* Check that it is not possible to select a loop duration + start time that violates the video duration

### CODE REVIEW

File: projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-panel/index.tsx [hunk 1]
```diff
@@ -417,10 +423,10 @@ export default function PosterPanel( {
...
 	const { poster, posterData } = attributes;
 
-	const pickPosterFromFrame = attributes?.posterData?.type === 'video-frame';
-	const previewOnHover = attributes?.posterData?.previewOnHover || false;
-	const previewAtTime = attributes?.posterData?.previewAtTime || posterData?.atTime || 0;
-	const previewLoopDuration = attributes?.posterData?.previewLoopDuration || DEFAULT_LOOP_DURATION;
+	const pickPosterFromFrame = posterData?.type === 'video-frame';
+	const previewOnHover = posterData?.previewOnHover || false;
+	const previewAtTime = posterData?.previewAtTime ?? posterData?.atTime ?? 0;
+	const previewLoopDuration = posterData?.previewLoopDuration ?? DEFAULT_LOOP_DURATION;
```
Comment (dhasilva) [inline projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-panel/index.tsx:429]: This was the cause of the issue with selecting `0`.

File: projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-panel/index.tsx [hunk 2]
```diff
@@ -320,6 +318,8 @@ function VideoFramePicker( {
 				max={ duration }
 				value={ timestamp }
 				wait={ 250 }
+				fineAdjustment={ 1 }
+				decimalPlaces={ 2 }
```
Comment (dhasilva) [inline projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-panel/index.tsx:322]: I added it here so all selectors have the same decimals, allowing for a more fine-tuned selection.

Comment (retrofox) [reply projects/packages/videopress/src/client/block-editor/blocks/video/components/poster-panel/index.tsx:322]: Fine-tuned selection is enabled even when it isn't shown by using UP/DOWN LEFT/RIGHT in the range control. Probably we'd like to keep polishing it

**Review by retrofox — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**